tag-historian/nitaghistorian.yml

```diff
@@ -0,0 +1,1019 @@
+      will respond with a *WebSocketQueryResponse* message once the query has been completed.
+    required: [action]
+    allOf:
+      - type: object
+        properties:
+          action:
+            description: The action to perform. Must be set to "TagHistorianQueryRequest".
+            type: string
+            enum: ['TagHistorianQueryRequest']
+            example: TagHistorianQueryRequest
+      - $ref: '#/definitions/HistoricalTagQuery'
+  WebSocketQueryResponse:
+    title: Web Socket Query Response
+    description: A web socket message sent by the server in response to a *WebSocketQueryRequest*
+      containing the result of the query.
+    type: object
+    properties:
+      action:
+        description: The action performed. Must be set to "TagHistorianQueryResponse".
+        type: string
+        enum: ['TagHistorianQueryResponse']
+        example: TagHistorianQueryResponse
+      tagHistorianValues:
+        $ref: '#/definitions/TagHistorianValues'
+  WebSocketAsyncQueryRequest:
+    title: Web Socket Async Query Request
+    description: A web socket message sent by the client to asynchronously query for historical tag
+      values. The server will respond with a *WebSocketAsyncQueryResponse* message confirming the start
+      of the query, a *WebSocketAsyncQueryData* message for each query as they complete, and finally
+      a *WebSocketAsyncQueryComplete* once all queries have completed.
+    type: object
+    required:
+      - action
+      - queries
+    properties:
+      action:
+        description: The action to perform. Must be set to "TagHistorianAsyncQueryRequest".
+        type: string
+        enum: ['TagHistorianAsyncQueryRequest']
+        example: TagHistorianAsyncQueryRequest
+      queries:
+        description: The list of history queries to perform.
+        type: array
+        items:
+          $ref: '#/definitions/HistoricalTagQuery'
+    example:
+      action: TagHistorianAsyncQueryRequest
+      queries:
+        - decimation: 1000
+          endTime: '9999-01-01T00:00:00Z'
+          sortOrder: ASCENDING
+          startTime: '0001-01-01T00:00:00Z'
+          paths: [system1.tag1]
+  WebSocketAsyncQueryResponse:
+    title: Web Socket Async Query Response
+    description: A web socket message sent by the server to confirm a *WebSocketAsyncQueryRequest* sent
+      by the client. A *WebSocketAsyncQueryData* message will be sent for each query as they complete
+      and a *WebSocketAsyncQueryComplete* once all queries have completed.
+    type: object
+    properties:
+      action:
+        description: The action performed. Must be set to "TagHistorianAsyncQueryResponse".
+        type: string
+        enum: ['TagHistorianAsyncQueryResponse']
+        example: TagHistorianAsyncQueryResponse
+      queryJobs:
+        description: A list containing the information about each started query.
```

Comment:
Could we flip "started query" to "query started"? If you agree, please implement throughout the doc.